### ISSUE TITLE
aRDP - icon is missing on Izzyondroid

### ISSUE THREAD

**Issue by inson1** [iiordanov/remote-desktop-clients#534]

**Comment by iiordanov** [comment]
Is it the launcher icon that is missing?

**Comment by iiordanov** [comment]
Oh, I don't maintain that repository. Is there a contact link for the repo?

**Comment by inson1** [comment]
@iiordanov I guess just ping IzzySoft
best would be Fastlane structure, but this is repo for more apps right? I dont know how to do that in that case, maybe Triple-T structure would be better
https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/

**Comment by iiordanov** [comment]
How do I ping IzzySoft? :)

**Comment by inson1** [comment]
@IzzySoft hey could you help us?

**Comment by IzzySoft** [comment]
> best would be Fastlane structure

I fully agree there. And I could see to make a PR to get it started, feeding in what is set up with my repo currently. After merge this can then be easily adjusted to what is felt needed.

> but this is repo for more apps right? I dont know how to do that in that case, maybe Triple-T structure would be better

Triple-T might *look* as if it were better suited in this case – but it's easily missed that Fastlane covers that, too. First, other than at F-Droid.org where the "entry point" is fixed, with my repo the "entry point" is configurable, so we'd be free to choose where to put it for each app. But second, we don't even need that as the Fastlane definition provides the means. Details can be found in my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) – more specifically in its [first comment](https://gitlab.com/-/snippets/1895688#locations) which deals with possible "standard locations". So to be compatible with the standard, I'd say the best match is `/metadata/<packageName>` as "entry-point" – i.e. `/metadata/com.iiordanov.freebVNC` for bVNC and `/metadata/com.iiordanov.freeaRDP` for aRDP. Inside that the locales with the descriptions and images.

@iiordanov do you want me to open a PR with a starter package for you? To fix this specific issue, you'd then only have to add the icons (all else can be done whenever you find suitable – e.g. adjusting descriptions if you think adjustments are needed, maybe adding per-release changelogs, etc). The big advantage of this as opposed to you just providing the icons here and me adding them manually is: with each release, my updater would check your Fastlane structures for changes as well and pull them along, so you'd be in control of the presentation.

> it should be the same for Izzyondroid

Except that IoD does not support Triple-T, but then is more flexible with Fastlane, yes :wink:
